src/main/java/fi/metatavu/soteapi/tasks/AbstractUpdateJob.java

```diff
@@ -28,7 +34,9 @@ public void run() {
     try {
       userTransaction.begin();
       
-      execute();
+      if (isEnabled()) {
+        execute();
+      }
       
       userTransaction.commit();
     } catch (Throwable ex) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Catch Exception instead of Throwable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1181)